<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaddCourse</class>
 <widget class="QWidget" name="WaddCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="CourseName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="courseID">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CourseCoef">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Course name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Course Coef:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Course ID:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCourse">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Course</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
